.github/workflows/cpp-post-commit.yaml

```diff
@@ -44,6 +44,8 @@ jobs:
       matrix:
         test-group: [
           {name: C++, cmd: ./tests/scripts/run_cpp_unit_tests.sh},
+          {name: ttnn examples and cpp tests, cmd: ./build/test/ttnn/unit_tests_ttnn && ./tests/scripts/run_ttnn_examples.sh},
```

Comment:
Can we move `run_ttnn_examples.sh` back into the ttnn post commit? That's a python test.